WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainSuggestionsViewController.swift

```diff
@@ -139,3 +141,36 @@ extension RegisterDomainSuggestionsViewController: NUXButtonViewControllerDelega
         self.navigationController?.pushViewController(controller, animated: true)
     }
 }
+
+// MARK: - DomainCreditRedemptionSuccessViewControllerDelegate
+
+extension RegisterDomainSuggestionsViewController: DomainCreditRedemptionSuccessViewControllerDelegate {
+
+    /// Handles the dismission of the epilogue screen at the end of a successful domain registration, along with popping to the domains dashboard
+    func continueButtonPressed() {
+
+        dismiss(animated: true) { [weak self] in
+            if let popController = self?.navigationController?.viewControllers.first(where: {
+                                                                                $0 is UIHostingController<DomainsDashboardView>
+            }) ?? self?.navigationController?.topViewController {
+                self?.navigationController?.popToViewController(popController, animated: true)
+            }
```

Comment:
Any chance we can implement this without having to "know" there's a `UIHostingController<DomainsDashboardView>` before this VC?

This approach will work for us now but I fear it'll make this component harder to use in the future.

Maybe we can call a completion callback and let the previous VC handle its own dismissal logic?